core/src/main/java/bio/terra/pearl/core/service/workflow/DataChangeRecordService.java

```diff
@@ -14,7 +16,7 @@ public class DataChangeRecordService extends ImmutableEntityService<DataChangeRe
     private EnrolleeService enrolleeService;
 
 
-    public DataChangeRecordService(DataChangeRecordDao dao, EnrolleeService enrolleeService) {
+    public DataChangeRecordService(DataChangeRecordDao dao, @Lazy EnrolleeService enrolleeService) {
         super(dao);
```

Comment:
Just curious about the change to @lazy

Reply:
Was needed to resolve a circular dependency.  IIRC it was EnrolleeService -> DataChangeRecordService -> AdminTaskService -> EnrolleeService.  Which are all dependencies that are needed, so we just have to specify the order they're loaded